R/find_bad_apples.R

```diff
@@ -2,23 +2,98 @@
 #'
```

Comment:
The very top of your function is currently:

`#' find_bad_apples Documentation`

It should be (http://r-pkgs.had.co.nz/man.html):

> The first sentence becomes the title of the documentation. That’s what you see when you look at help(package = mypackage) and is shown at the top of each help file. It should fit on one line, be written in sentence case, but not end in a full stop.

Can you update to reflect this?

Reply:
I actually wanted "find_bad_apples Documentation" to be the title of the documentation. Should it be more descriptive like "Find outliers"?

Reply:
Yeah you don't want to use the same name as the function (e.g. with underscores and all), and you don't need to call it Documentation. I think just using "Find outliers" works. See this example from Hadley Wickham in the link I sent.
```R
#' Add together two numbers.
#' 
#' @param x A number.
#' @param y A number.
#' @return The sum of \code{x} and \code{y}.
#' @examples
#' add(1, 1)
#' add(10, 1)
add <- function(x, y) {
  x + y
}
```